src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -2668,7 +2668,7 @@ void UpdateMemoryCost(in UInt256 position, in UInt256 length)
                             EndInstructionTraceError(EvmExceptionType.BadInstruction);
                             return CallResult.InvalidInstructionException;
                         }
-                    case Instruction.SELFDESTRUCT:
+                    case Instruction.SELFDESTRUCT | Instruction.SENDALL: // SELFDESTRUCT is depricated and now this OP code is called SENDALL.
```

Comment:
Does this produce same IL as
case Instruction.SELFDESTRUCT:
case Instruction.SENDALL:

Reply:
You cannot do case and case because both values are the same. The compiler will not allow it.

Reply:
Ah got it so this is fake bitwise OR.